processes/security_team_members.md

```diff
@@ -13,27 +13,35 @@ and must be approved by current team members.
 Members of the security teams should indicate that they accept the privacy
 policies by PRing their acceptance to this file.
 
-## Team that triages security reports against node core
+## Team that triages security reports against Node.js
 
-- @cjihrig - **Colin Ihrig**
-- @indutny - **Fedor Indutny**
-- @jasnell - **James M Snell**
-- @mcollina - **Matteo Colina**
-- @mhdawson - **Michael Dawson**
-- @MylesBorins - **Myles Borins**
-- @rvagg - **Rod Vagg**
-- @vdeturckheim - **Vladimir de Turckheim**
+TODO sync with nodejs-private/triage=team, nodejs/triage-team
+
+The [TSC](https://github.com/nodejs/node#tsc-technical-steering-committee)
+are all members of the Triage Team.
+
+These non-TSC and TSC Emeriti are Triage Team members:
+* [indutny](https://github.com/indutny) -
+**Fedor Indutny** &lt;fedor.indutny@gmail.com&gt;
+* [rvagg](https://github.com/rvagg) -
+**Rod Vagg** &lt;rod@vagg.org&gt;
+- [vdeturckheim](https://github.com/vdeturckheim) -
+**Vladimir de Turckheim** &lt;vladimir@sqreen.io &gt;
+
+List is from the [member page](https://hackerone.com/nodejs/team_members) for
+the Node.js program on HackerOne.
 
 ### Emeritus
 
 - @bnoordhuis - **Ben Noordhuis**
 - @jasnell - **James M Snell**
 - @shigeki - **Shigeki Ohtsu**
 
-List is from ["security" alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
 
 ## Team with access to security issues
 
+XXX Unclear what meaning of this is now that we use H1. Almost certainly not up to date.
```

Comment:
+1 for removing it. It was probably useful when we were first figuring things out, but yeah, keeping multiple lists in multiple places is going to pretty much guarantee stuff gets out of sync at some point and then you don't know which one is the source of truth.